lib/jnpr/junos/facts/get_software_information.py

```diff
@@ -17,6 +17,12 @@ def _get_software_information(device):
             except Exception:
                 pass
         try:
+            # JDM for Junos Node Slicing
+            return device.rpc.cli("show version all-servers", format='xml',
```

Comment:
" I hadn't used the RPC because | display xml rpc didn't show it." This is a long pending bug.